app/models/page_content_register.rb

```diff
@@ -17,7 +19,10 @@ def new_blocks_attributes
                         else
                           @new_blocks_attributes
                         end
-    return @new_blocks_attributes if home_page? || footer?
+
+    if home_page? || footer?
+      return blocks_attributes + srcset_blocks(blk_attrs: blocks_attributes)
```

Comment:
@tomas-stefano _new_srcset_blocks_ adds new block attributes which contain _srcset_ attributes for each image on the homepage. It's injected into the _new_blocks_attributes_ if they do not exist.